docs/getting-started/development.md

```diff
@@ -234,10 +110,12 @@ Once complete, the following services will be available to you:
 * VNC for the Selenium Chrome node --- `192.168.120.105:5999`
 * VNC for the Selenium Firefox node --- `192.168.120.110:5999`
 
-Then ensure that all of the Behat-related options are present in your Moodle `config.php` (see the recommended configuration for advice) and run the following command to bootstrap your test site:
+### Behat
+
+Review the Behat-related options in your Moodle `config.php` (see the recommended configuration for advice) and run the following command to bootstrap the testing environment:
 
 ```
-$ vagrant ssh app-debug-1 --command 'php current/admin/tool/behat/cli/init.php'
+$ vagrant ssh app-debug-1 --command 'php ~/releases/vagrant/admin/tool/behat/cli/init.php'
```

Comment:
This change should no longer be necessary after https://github.com/LukeCarrier/moodle-ubiquitous/commit/f77add6508de746fca62a4c393daf62e058020f7. #12 tracks the permanent solution of just setting it via the Salt states.